Luke.md

```diff
@@ -1,35 +1,34 @@
-## 08/03 - 08/07 ##
+## 08/10 - 08/14 ##
 
 ### Last Week ###
 
+* [TV remote control]
+    - Proof of concept
+        - discussed with Eric
+        - identified the scenario
+        - surveyed pros and cons of Ajax and WebSocket
+
 * [System Merge]
-    - [WIP] Bug 1179606 - [System] disable mobileid related modules
-        - investigated
-    - [Done] Bug 1179605 - [System] disable telephony related modules
-        - verified and closed
-    - [Done] Bug 1179614 - [System] disable NFC related modules
-        - verified and closed
     - [Done] Bug 1156596 - [System] Support device type in base module
-        - Ricky is working on this
-        - discussed with Ricky about its implementation
-
-* [System App]
-    - [Done] Bug 1191671 - clearing previous search in rocketbar should not dismiss keyboard
-        - helped identify the regression
+        - helped on reviewing the patch
+    - [WIP] Bug 1179606 - [System] disable mobileid related modules
+        - WIP but suspended temporarily
 
 * [Keyboard App]
     - [WIP] Bug 1094833 - Autocorrect should not correct 'lol' to 'lil'
-        - made a simple patch but got r-
-
-* [MultiScreen]
-    - helped test the boundry issue on TV
-        - https://github.com/luke-chang/gaia/tree/multiscreen_private
+        - encountered an unexpected problem and waiting for feedback
 
 ### This Week ###
 
+* [TV remote control]
+    - Proof of concept
+        - write a demo page to test the performance between Ajax and WebSocket
+
+* [SUPL NI]
+    - [WIP] Bug 1195602 - [GPS] Support SUPL NI case in Gaia
```

Comment:
It's about time to land it on v2.2 as well as unit tests since gecko part has gotten r+ last week.